### PR TITLE
Ecc 1001

### PR DESCRIPTION
**Pull request by NandanaRaghu**

### CODE REVIEW

File: test/unit/controllers/YouCannotChangeAddressControllerSpec.scala [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+import org.mockito.ArgumentMatchers.any
+import org.mockito.Mockito.when
+import play.api.mvc.Result
+import play.api.test.Helpers.{LOCATION, _}
+import uk.gov.hmrc.auth.core.AuthConnector
+import uk.gov.hmrc.eoricommoncomponent.frontend.controllers.{
+  AreYouSureYouWantToDeleteVatController,
+  YouCannotChangeAddressController
+}
+import uk.gov.hmrc.eoricommoncomponent.frontend.forms.models.VatEUDetailsModel
```
Comment (vigneshwaripandi) [inline test/unit/controllers/YouCannotChangeAddressControllerSpec.scala:30]: Please use unused imports

File: test/unit/views/YouCannotChangeAddressOrganisationSpec.scala [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package unit.views
+
+import org.jsoup.Jsoup
+import org.jsoup.nodes.Document
+import play.api.test.Helpers.contentAsString
+import uk.gov.hmrc.auth.core.AffinityGroup.{Agent, Organisation}
```
Comment (vigneshwaripandi) [inline test/unit/views/YouCannotChangeAddressOrganisationSpec.scala:22]: Same here unused imports needs to be removed

File: test/unit/controllers/YouCannotChangeAddressControllerSpec.scala (hunk 1 above)
Comment (NandanaRaghu) [reply test/unit/controllers/YouCannotChangeAddressControllerSpec.scala]: removed unused imports

File: test/unit/views/YouCannotChangeAddressOrganisationSpec.scala (hunk 2 above)
Comment (NandanaRaghu) [reply test/unit/views/YouCannotChangeAddressOrganisationSpec.scala]: removed unused imports

**Review by vigneshwaripandi — DISMISSED** [review]
LGTM

**Review by vigneshwaripandi — APPROVED** [review]
LGTM

**Pull request merged**